edgelet/iotedge/test-files/config/ca-quickstart/keyd.toml

```diff
@@ -11,10 +11,10 @@ device-id = "file:///var/secrets/aziot/keyd/device-id"
 uid = 5556
 keys = ["aziot_identityd_master_id", "device-id"]
 
-[[principal]]
```

Comment:
why this change in this file, and the files below?

Reply:
In order to add "aziot-edged-ca-temp" to all files cleanly, I had to rearrange some of the output ordering. That causes this principal to come before the previous. Ordering of these principals has no effect at runtime though.